### ISSUE TITLE
an argument of type `std::option::Option<Reveal>` is missing

### ISSUE THREAD

**Issue by Goosie** [LNP-BP/nodes#18]
A second warning comes by after running ./install.sh
  Compiling rgb-cli v0.8.0-rc.1
error[E0061]: this function takes 4 arguments but 3 arguments were supplied
   --> /home/rgbalice/.cargo/registry/src/github.com-1ecc6299db9ec823/rgb-cli-0.8.0-rc.1/src/command.rs:279:41
    |
279 |                     let status = client.consume_transfer(consignment, force, progress)?;
    |                                         ^^^^^^^^^^^^^^^^                     -------- an argument of type `std::option::Option<Reveal>` is missing
    |
note: associated function defined here
   --> /home/rgbalice/.cargo/registry/src/github.com-1ecc6299db9ec823/rgb_rpc-0.8.1/src/client.rs:260:12
    |
260 |     pub fn consume_transfer(
    |            ^^^^^^^^^^^^^^^^
help: provide the argument
    |
279 |                     let status = client.consume_transfer(consignment, force, /* std::option::Option<Reveal> */, progress)?;
    |                                                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

For more information about this error, try `rustc --explain E0061`.
error: could not compile `rgb-cli` due to previous error
error: failed to compile `rgb-cli v0.8.0-rc.1`, intermediate artifacts can be found at `/tmp/cargo-install6AIY4I`
  Downloaded storm-cli v0.8.0

**Comment by crisdut** [comment]
Hi @Goosie,

Here: https://github.com/LNP-BP/nodes/pull/20

I updated all libs to correct version. 

Please, try again.

**Comment by dr-orlovsky** [comment]
I merged #20. Was this issue fixed?

**Comment by Goosie** [comment]
Well following the instruction in the readme I'm still gett an error:
I do:
git clone https://github.com/LNP-BP/nodes.git
cd repos/nodes
rustup update nightly
cargo +nightly install lnpbp_nodes -Z bindeps

Gives the error:
error: could not find `lnpbp_nodes` in registry `crates-io` with version `*`

**Comment by dr-orlovsky** [comment]
Yes, since the original intent I had with doing this crate has failed: rust cargo package manager doesn't allow to publish crates made of binaries of other crates. This crate was created to simplify the installation of all LNP/BP nodes - which reside as binaries in their own crates - and cargo prohibits exactly that. So there is no way to solve that issue; the only possible approach is to have a docker containers and automation on building nodes into them. Related discussion: https://github.com/LNP-BP/docker/issues/27#issuecomment-1366826580
